README.md

```diff
@@ -11,10 +11,11 @@ The AEP Consent Collection mobile extension enables consent preferences collecti
 Integrate the Consent extension into your app by including the following in your gradle file's `dependencies`:
 
 ```
-implementation 'com.adobe.marketing.mobile:edgeconsent:1.+'
-implementation 'com.adobe.marketing.mobile:edge:1.+'
-implementation 'com.adobe.marketing.mobile:core:1.+'
+implementation 'com.adobe.marketing.mobile:edgeconsent:2.0.0'
```

Comment:
Can you make this same change in `getting-started.md`, lines 30-33 which list the dependencies? 
Can you also include the link to all the current versions?

> Add the Mobile Core and Edge extensions to your project using the app's Gradle file. See the current version list for the latest extension versions.